WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -222,11 +222,16 @@ - (BlogDetailsSection *)generalSectionViewModel
                                                  }]];
 
     if ([Feature enabled:FeatureFlagPlans] && [self.blog supports:BlogFeaturePlans]) {
-        [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Plans", @"Action title. Noun. Links to a blog's Plans screen.")
-                                                        image:[UIImage imageNamed:@"icon-menu-plans"]
-                                                     callback:^{
-                                                         [weakSelf showPlans];
-                                                     }]];
+        BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Plans", @"Action title. Noun. Links to a blog's Plans screen.")
+                                                              image:[UIImage imageNamed:@"icon-menu-plans"]
+                                                           callback:^{
+                                                               [weakSelf showPlans];
+                                                           }];
+
+        Plan *plan = [[Plan alloc] initWithPlan:[self.blog.planID integerValue]];
```

Comment:
Interesting, if planID is nil/0, this doesn't crash but initializes to `Free`. I'm guessing the compiler is picking the first enum case as the default instead of crashing, but it's still important to handle the invalid case: we don't want to show Free for a Business site just because we haven't synced the site's plan yet.